content/help/faq/security/accessing-non-istio-services.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to disable mutual TLS on clients to access the Kubernetes API Server (or any services that don't have Istio sidecar)?
```

Comment:
Let's keep the title shorter, otherwise it doesn't look good when rendered. How about just:

How can services that use Istio access non-Istio services?

Reply:
Done